build/build.js

```diff
@@ -0,0 +1,112 @@
+const rollup = require('rollup');
```

Comment:
any chance the build files get written in typescript and executed via ts-node in the package.json? It feels a bit disappointing having to mix these parts as JS in the repo. Also we can't reuse the linter since we'd have to add eslint additionally and mix/match the rule configuration.

Reply:
Sounds nice, I'll refactor this bit